packages/nft-analytics/src/models/valuation.ts

```diff
@@ -0,0 +1,68 @@
+import { AdapterError } from '@chainlink/ea-bootstrap'
+
+import { BitscrunchValuationsGetPayload } from '../api/bitscrunch/types'
+import { Provider } from '../api/constants'
+
+export interface IValuation {
+  valuation: {
+    priceEstimate: {
+      currency: string
+      amount: number
+      units: string
+    }
+    priceEstimateUpperBound: {
+      currency: string
+      amount: number
+      units: string
+    }
+    priceEstimateLowerBound: {
+      currency: string
+      amount: number
+      units: string
+    }
+  }
+}
+
+const transformer = {
+  [Provider.BITSCRUNCH]: (payload: BitscrunchValuationsGetPayload): IValuation => {
+    return {
+      valuation: {
+        priceEstimate: {
+          currency: payload.price_estimate.unit,
+          amount: payload.price_estimate.value,
```

Comment:
Important: format it as `wei` and make sure it is a `string` (as `floorprices` endpoint)

Also include `network` and `timestamp` even though they have `null` values. Can't any of them have a value (just curiosity)?